src/main/public/stylesheets/components/_form.scss

```diff
@@ -27,3 +27,32 @@
   padding: 5px 4px 4px;
   text-transform: none;
 }
+
+.triple-column {
```

Comment:
Is that reusable? If this is something specific to SoM we might want to rename it to accordingly like `statement-of-means-columns-container`

Reply:
I don't know it's copy&paste from prototype. Code was written by Tarik.

Reply:
I agree with @geek-soft -- I think this should be SoM specific, indicated either by name or should be placed in a separate SASS file.

Reply:
Well I belive the name and usage is so generic we will use it in near future in other places

Reply:
I have sent it to Tarik with question if this is for SoM-only

Reply:
@tarik-johnston1

Reply:
ok, I'll refactor it